platform/ethereum/blockbook/model.go

```diff
@@ -73,4 +73,5 @@ type EthereumSpecific struct {
 	GasLimit *big.Int `json:"gasLimit"`
 	GasUsed  *big.Int `json:"gasUsed"`
 	GasPrice string   `json:"gasPrice"`
+	Data     string   `json:"data,omitempty"`
```

Comment:
It could be empty?

Reply:
Yes, this field is added recently, we may switch to old node which doesn’t return data